storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11961,23 +11966,31 @@ static void rocksdb_set_max_background_jobs(THD *thd,
   RDB_MUTEX_UNLOCK_CHECK(rdb_sysvars_mutex);
 }
 
+static int rocksdb_validate_update_cf_options(THD *thd,
+                                              struct st_mysql_sys_var *var,
+                                              void *save,
+                                              st_mysql_value *value) {
+  // just disallow setting to null; otherwise same as default string checker
+  char buff[STRING_BUFFER_USUAL_SIZE];
+  const char *str;
+  int length;
+
+  length= sizeof(buff);
+  if ((str= value->val_str(value, buff, &length)))
+    str= thd->strmake(str, length);
+  *(const char**)save= str;
```

Comment:
Should we be setting save to str if str is nullptr?

Reply:
I'm not sure. Maybe let's discuss my high-level question first so we can potentially avoid worrying about when to save it.